pyproject.toml

```diff
@@ -1,3 +1,13 @@
+[tool.autopep8]
```

Comment:
Adding a blocking comment to wait for the TH release.

Reply:
Marking resolved.